Makefile

```diff
@@ -13,6 +13,7 @@ CFLAGS += -D DEFAULT_STACK_ADDR=0xFFFFF000
 
 OBJS_EXT :=
 
+EXT_LIST :=
```

Comment:
To avoid potential confusion, let's rename it to `RVEXT_LIST` instead of `EXT_LIST`. The latter may mislead someone into thinking it represents a list of file extensions.